microsite/sidebars.ts

```diff
@@ -238,7 +238,7 @@ export default {
           'features/techdocs/techdocs-overview',
           'features/techdocs/getting-started',
           'features/techdocs/concepts',
-          'features/techdocs/addons',
```

Comment:
Can you spin up the doc site to test this? won't this change the TechDocs Addons sidebar entry to TechDocs Addons--new?

Wondering what it does to this section:

![image](https://github.com/user-attachments/assets/e105cde1-1731-4e65-8cb3-bd0a9f445651)